book/06-github/sections/1-setting-up-account.asc

```diff
@@ -38,7 +38,7 @@ Then select the ``SSH keys'' section along the left-hand side.
 .The ``SSH keys'' link.
 image::images/ssh-keys.png[The ``SSH keys'' link.]
 
-From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key''.
+From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key.''
```

Comment:
This is an example that always seemed ambiguous to me. We're telling the user to look for a specific button that reads:

> Add key

…without punctuation. They're not looking for a button with a period. So when we quote something like that, it seems better to put the exact content of the button inside the quotes, just like you would with `monospace` - the period doesn't go inside the `single ticks.`